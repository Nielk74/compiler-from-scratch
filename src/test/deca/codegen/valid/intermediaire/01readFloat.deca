{
    float num;
    num = readFloat();
    println(num);
}