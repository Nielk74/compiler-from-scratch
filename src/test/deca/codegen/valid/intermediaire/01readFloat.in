5.6
67.89