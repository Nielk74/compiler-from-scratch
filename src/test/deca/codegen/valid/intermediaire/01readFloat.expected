5.60000e+00
